src/components/compound/DlSearches/DlSmartSearch/components/DlSuggestionsDropdown.vue

```diff
@@ -90,7 +90,7 @@ export default defineComponent({
             handleOption(value)
         }
         const emitModelValue = (event: any) => {
-            emit('update:modelValue', event)
+            emit('update:model-value', event)
```

Comment:
@prodan-eric we have in the `use-model-toggle.ts` hardcoded event type for vue2 or vue3. can you check why we need it there and see if its causing issues for us anywhere in the future?

Reply:
I checked it and changing the caseing inside that file won't fix the problem with vue 2, since it seems to necessarily want kebab-case. I think it's best to stick to the "model-value"  synthax everywhere as it works both on vue 2 and vue 3

Reply:
should we change the casing in the file to model-value always ? @prodan-eric or would that cause issues with vue3 ?